File: 3005. Count Elements With Maximum Frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int maxFreq = 0;
        int[] freq = new int[101];
        for (int i : nums) {
            freq[i]++;
            maxFreq = Math.max(maxFreq, freq[i]);
        }
        int answer = 0;
        for (int i : freq) {
            if (i == maxFreq) answer += i;
        }
        return answer;
    }
}

File: 3005. Count Elements With Maximum Frequency.go
func maxFrequencyElements(nums []int) int {
    freq := [101]int{}
    maxFreq := 0
    for _, i := range(nums) {
        freq[i] += 1
        maxFreq = max(maxFreq, freq[i])
    }
    count := 0
    for _, i := range(nums) {
        if freq[i] == maxFreq {
            count++
        }
    }
    return count
}
